mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -300,3 +301,25 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
 		ARG 3 width
 		ARG 4 height
 		ARG 5 color
+	METHOD method_49696 createDivider (II)Lit/unimi/dsi/fastutil/ints/IntIterator;
+		ARG 0 sideLength
+		ARG 1 textureSideLength
+	METHOD method_49697 drawNineSlicedTexture (Lnet/minecraft/class_4587;IIIIIIIIII)V
+		ARG 0 matrices
+		ARG 1 x
+		ARG 2 y
+		ARG 3 width
+		ARG 4 height
+		ARG 5 sideSliceWidth
+		ARG 6 sideSliceHeight
+		ARG 7 centerSliceWidth
+		ARG 8 centerSliceHeight
+		ARG 9 u
+		ARG 10 v
+	METHOD method_49698 setScissor (Lnet/minecraft/class_8030;)V
+		ARG 0 rect
+	CLASS class_8214 ScissorStack
+		FIELD field_43099 stack Ljava/util/Deque;
+		METHOD method_49699 pop ()Lnet/minecraft/class_8030;
+		METHOD method_49700 push (Lnet/minecraft/class_8030;)Lnet/minecraft/class_8030;
+			ARG 1 rect
```

Comment:
Hmm, is FocusedRect the correct name for this type now? I don't think there's anything particularly "focused" about scissored areas.